v22.2/sso-db-console.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Single Sign-on for DB Console
+title: Single Sign-on (SSO) for DB Console
```

Comment:
For the CockroachDB Cloud version of this page, the "on" is capped, not sure what we prefer style-wise, but it'd be great to standardize these.

Reply:
Done.